### ISSUE TITLE
NavigationController entry with different controllers height 

### ISSUE THREAD

**Issue by Banck** [huri000/SwiftEntryKit#184]
@huri000 Hello!
Is it possible to have different height for different controller when display Navigation controller as entry?

**Comment by huri000** [comment]
I haven't tested this specific use case, but I don't see why wouldn't it be possible. 
Have you happened to test it?

**Comment by Banck** [comment]
Hm...I thought ViewController entry doesn't work with `attributes.positionConstraints.size = .. height: .intrinsic`. As I understand for ViewController entry we need to specify height for size, doesn't it? If no, then how can I create entry with navigation controller, when some controller has 100 height and another one, for example 200?

**Comment by huri000** [comment]
You can control it using the navigation controller itself, for example, by "installing" a method that changes the height of its `view` animately. Thus also changing the entire subview hierarchy's height.
You want each view controller to decide its own height, right? So I would access `navigationController` in `viewWillAppear(_:)` of each view controllers embedded within the navigation controller and call the animating method with the intended height value.

I prepared some example for you showing exactly what I meant. Just replace `ExampleNavigationController` folder with what I gave you and run the example project with the navigation sample preset: [tap me](https://github.com/huri000/SwiftEntryKit/files/3195104/ExampleNavigationController.zip).

Basically, your question is not directly related to `SwiftEntryKit` 
It is more about a specific use case. 
So I'm closing this for now.
